### PR TITLE
Feat/download report

### PR DESCRIPTION
**Pull request by adipaul1981**
closes [SJIP-131](https://d3b.atlassian.net/browse/SJIP-131)
closes [SJIP-128](https://d3b.atlassian.net/browse/SJIP-128)

image

image

### CODE REVIEW

File: src/store/report/types.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// export interface ReportState {
```
Comment (oliviercperrier) [inline src/store/report/types.ts]: Remove?

File: src/views/DataExploration/components/tabs/Biospecimens/index.tsx [hunk 2]
```diff
@@ -16,11 +16,15 @@ import { useUser } from "store/user";
 import { updateUserConfig } from "store/user/thunks";
 
 import styles from "./index.module.scss";
+import { Button } from "antd";
```
Comment (oliviercperrier) [inline src/views/DataExploration/components/tabs/Biospecimens/index.tsx:19]: move before style

File: src/views/DataExploration/components/tabs/Biospecimens/index.tsx [hunk 3]
```diff
@@ -150,6 +155,12 @@ const BioSpecimenTab = ({
               },
             })
           ),
+        extra: [
+          <Button onClick={(e) => downloadReport(ReportType.BIOSEPCIMEN_DATA)}>
```
Comment (oliviercperrier) [inline src/views/DataExploration/components/tabs/Biospecimens/index.tsx:159]: Checker si on veut un button type="primary"

File: src/views/DataExploration/components/tabs/Participants/index.tsx [hunk 4]
```diff
@@ -1,34 +1,27 @@
...
-import { useUser } from "store/user";
+import {ProColumnType} from "@ferlab/ui/core/components/ProTable/types";
+import {getProTableDictionary} from "utils/translation";
+import {Button, Dropdown, Menu, Tag} from "antd";
+import {useDispatch} from "react-redux";
+import {updateUserConfig} from "store/user/thunks";
+import {useUser} from "store/user";
 
 import styles from "./index.module.scss";
+import {ReportType} from "services/api/reports/models";
```
Comment (oliviercperrier) [inline src/views/DataExploration/components/tabs/Participants/index.tsx:17]: Move before style

**Review by oliviercperrier — CHANGES_REQUESTED** [review]

File: src/views/DataExploration/components/PageContent/index.tsx [hunk 5]
```diff
@@ -112,10 +83,31 @@ const PageContent = ({
...
+  const onRowSelection = (selectedRowKey: Key, type: TAB_IDS) => {
+    selectedRows.includes(selectedRowKey)
+      ? setSelectedRows(selectedRows.filter((e) => e !== selectedRowKey))
+      : setSelectedRows([...selectedRows, selectedRowKey]);
+
+    setSelectType(type);
+  };
+
+  const onAllRowSelection = (selectedRowKeys: Key[], type: TAB_IDS, selected: boolean) => {
+    //todo fix this
```
Comment (oliviercperrier) [inline src/views/DataExploration/components/PageContent/index.tsx:99]: Still need fix?

**Review by oliviercperrier — CHANGES_REQUESTED** [review]

**Review by oliviercperrier — APPROVED** [review]

File: src/views/DataExploration/components/tabs/Biospecimens/index.tsx (hunk 3 above)
Comment (adipaul1981) [reply src/views/DataExploration/components/tabs/Biospecimens/index.tsx:168]: according the the figma done by Lucas. No its not a primary.

**Pull request merged**